lib/src/file-input/FileInput.tsx

```diff
@@ -308,6 +310,7 @@ const Label = styled.label`
   font-size: ${(props) => props.theme.labelFontSize};
   font-weight: ${(props) => props.theme.labelFontWeight};
   line-height: ${(props) => props.theme.labelLineHeight};
+  ${(props) => !props.helperText && `margin-bottom: 0.5rem;`}
 `;
```

Comment:
Same here. In the specs these margins are shown as 0.25 rem. Is this change really needed and the specs should be updated?
![image](https://user-images.githubusercontent.com/10975289/171833523-0d4362a8-f678-46c4-8a29-709a8b851b8f.png)